microsoft-edge/devtools-guide-chromium/whats-new/2021/05/devtools.md

```diff
@@ -0,0 +1,228 @@
+
+Perform the following steps to launch your project. These steps assume you are in Visual Studio Code, with no folder opened.
+
+1.  In the **Activity Bar** on the left, select the **Microsoft Edge Tools** button.  In the **Side Bar**, the **Microsoft Edge Tools: Targets** panel appears, showing a single button (**Launch Instance**).
+
+    :::image type="complex" source="../../media/2021/05/edge-tools-extension-in-context-docs-initial-side-bar.msft.png" alt-text="The Microsoft Edge Tools: Targets panel in the Side Bar of Visual Studio Code, showing the Launch Instance button" lightbox="../../media/2021/05/edge-tools-extension-in-context-docs-initial-side-bar.msft.png":::
+       The **Microsoft Edge Tools: Targets** panel in the **Side Bar** of Visual Studio Code, showing the **Launch Instance** button
+    :::image-end:::  
+
+1.  In the **Microsoft Edge Tools: Targets** panel, select the **open a folder** link, and open your project folder.  The **Microsoft Edge Tools: Targets** panel now shows two buttons: **Launch Instance** and **Generate launch.json**.
+
+    :::image type="complex" source="../../media/2021/05/edge-tools-extension-in-context-docs-side-bar.msft.png" alt-text="The Microsoft Edge Tools: Targets panel in the Side Bar of Visual Studio Code with the Launch Instance and Generate launch.json buttons" lightbox="../../media/2021/05/edge-tools-extension-in-context-docs-side-bar.msft.png":::
+       The **Microsoft Edge Tools: Targets** panel in the **Side Bar** of Visual Studio Code with the Launch Instance and Generate launch.json buttons
+    :::image-end:::  
+    
+1.  In the **Side Bar**, select the **Generate launch.json** button.  A `launch.json` file is generated in the root of the project, and is opened for editing.
+
+    :::image type="complex" source="../../media/2021/05/edge-tools-extension-in-context-docs-launch-json-file.msft.png" alt-text="Editing the URL in the launch.json file" lightbox="../../media/2021/05/edge-tools-extension-in-context-docs-launch-json-file.msft.png":::
+       Editing the URL in the `launch.json` file
+    :::image-end:::
+
+1.  In the `launch.json` file, change the URL from an empty string to a URL for your project.  Or leave the string empty, to show the documentation page.
+
+1.  Save the `launch.json` file.
+
+1.  In the **Side Bar**, select the **Launch Project** button.  Microsoft Edge opens, showing the URL that you specified.  Also, in Visual Studio Code, DevTools opens.
+
+
+
+
+
+## Download the Microsoft Edge preview channels  
+
+If you are on Windows, Linux, or macOS, consider using the [Microsoft Edge preview channels][MicrosoftEdgePreviewChannels] as your default development browser.  The preview channels give you access to the latest DevTools features.  
+
+
+## Getting in touch with Microsoft Edge DevTools team  
+
+[!INCLUDE [contact DevTools team note](../../includes/contact-whats-new-note.md)]
```

Comment:
I think we need to update this note and remove the original page and author links. We can add them back when we actually have announcements from the Chromium project

Reply:
File "contact-whats-new-note.md" doesn't exist in this repo.

Reply:
@Reezaali - need your clarification of what actions to do re: the include.